DiverseSelector/diversity.py

```diff
@@ -68,12 +69,13 @@ def compute_diversity(
         "logdet": logdet,
         "shannon_entropy": shannon_entropy,
         "wdud": wdud,
-        "total_diversity_volume": total_diversity_volume,
         "gini_coefficient": gini_coefficient,
     }
 
     if div_type in func_dict:
         return func_dict[div_type](features)
+    elif div_type == "hypersphere_overlap_of_subset":
+        return hypersphere_overlap_of_subset(library, features)
```

Comment:
The usage of `features` is a little bit confusing here. Maybe try `feature_subset`? Someone would wonder what are `library` and `features` here. Moreover, one may assume the features denotes the full dature matrix. Anything that makes it clear works.